dev/com.ibm.ws.app.manager.springboot/.classpath

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="src" output="bin" path="src"/>
-	<classpathentry kind="src" output="bin_test" path="test"/>
+	<classpathentry kind="src" output="build/classes/java/main" path="src"/>
+	<classpathentry kind="src" output="build/classes/java/test" path="test"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry kind="output" path="build"/>
```

Comment:
Easy enough to change these back.  But, having the paths be the same for eclipse and gradle not work?  That removes .ignore entries, and simplifies cleaning up.

Reply:
The gradle and eclipse output directories are purposely in different directories for every projects in open liberty.  This allows for both to be able to build at the same time without interfering with each other.  The .gitignore files in the project directories can mostly be removed because the root level .gitignore file already has `/*/*/bin/` and `/*/*/build/`

Reply:
Fixed.